docs/source/get_started/modular_design.rst

```diff
@@ -28,6 +28,12 @@ This class can contain learnable parameters and `NaN` (missing value) handling.
 It takes :class:`~torch_frame.data.TensorFrame` as input and applies stype-specific feature encoder (specified via :obj:`stype_encoder_dict`) to :obj:`~torch.Tensor` of each stype to get embeddings for each :obj:`~torch_frame.stype`.
 The embeddings of different :obj:`stypes<torch_frame.stype>` are then concatenated to give the final 3-dimensional :obj:`~torch.Tensor` :obj:`x` of shape :obj:`[batch_size, num_cols, channels]`.
 
+.. note::
+    Different :obj:`stypes<torch_frame.stype>` can have the same internal representation, e.g. both :class:`~torch_frame.embedding` and :class:`~torch_frame.text_embedded` are stored with :class:`~torch_frame.data.MultiEmbeddingTensor`.
```

Comment:
Can we expand a bit why this is the case? people may not immediately understand `MultiEmbeddedTensor`. I'd just say they both generate embeddings in tensor frame.